pydoc/templates/sphinx/conf.py.tmpl

```diff
@@ -16,6 +16,9 @@ release = version
 
 html_theme = 'sphinx_rtd_theme'
 html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+html_theme_options {
+    'analytics_id':  'UA-17997319-4',
+}
```

Comment:
This still feels like the right integration point to me. I don't think it is something that should require futzing around with extra theme files and overwriting `layout.html` to do. We should talk about it in the theme ticket.